### ISSUE TITLE
Flake8 rule 'E901' reported an invalid point

### ISSUE THREAD

**Issue by blwy10** [AtomLinter/linter-flake8#398]
Flake8 reported an invalid point for the rule `E901`, with the messge `SyntaxError: invalid syntax`.

Code below:
```
"""Extract data into csv."""


import re
import csv


def main():
    """Main function."""
    regex = re.compile(r'\d\s(\d+)\s([^£]+)£\s([0-9.]+)\s£\s([0-9.]+)',
                       re.UNICODE)

    with open('raw_data.txt')


if __name__ == '__main__':
    main()
```

Debug information:
Atom version: 1.15.0
Flake8 version: `2.5.1 (pep8: 1.7.0, mccabe: 0.3.1, pyflakes: 1.0.0) CPython 2.7.10 on Windows`

**Comment by Arcanemagus** [comment]
Please update your `flake8` version (`pip install -U flake8`). This particular error should be fixed in `flake8>=3.2.1`.

**Comment by Arcanemagus** [comment]
Closing due to a lack of response, if you are still having issues after updating `flake8` please provide code to reproduce this!
